### PR TITLE
enable request for KYC details

### LINKED ISSUE #77: Feature Request: ID Search & Verification
**Is your feature request related to a problem? Please describe.**
This web service enables your application to query the various registrar of persons in the various countries in East Africa. Currently available for Kenya and Rwanda only [ID Search & Verification](https://developer.jengaapi.io/reference#identity-verification)

### PR DESCRIPTION
**Pull request by MidigoF**
## What is the Purpose?
Enable request for KYC details 

## Issue(s) affected?
Fixes #77

### CODE REVIEW

**Comment by MidigoF** [comment]
I'm still experiencing this error 
`{:error,
 %{
   message: %{
     "fault" => %{
       "detail" => %{
         "errorcode" => "messaging.adaptors.http.flow.ApplicationNotFound"
       },
       "faultstring" => "Unable to identify proxy for host: jenga and url: /customer-test/v2/identity/verify"
     }
   },
   status: 404
 }}`

File: lib/ex_pesa/Jenga/kyc/kyc.ex [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            lastName: _lastName,
+            dateOfBirth: _dateOfBirth,
+            documentNumber: documentNumber,
+            countryCode: countryCode
+          }
+        } = requestBody
+      ) do
+    message = "#{merchant_code()}#{documentNumber}#{countryCode}"
+
+    make_request("/customer-test/v2/identity/verify", requestBody, [
```
Comment (manuelgeek) [inline lib/ex_pesa/Jenga/kyc/kyc.ex:24]: @MidigoF change url to `/customer/v2/identity/verify`

File: lib/ex_pesa/Jenga/kyc/kyc.ex [hunk 2]
```diff
@@ -0,0 +1,32 @@
+defmodule ExPesa.Jenga.KYC do
+  @moduledoc """
+  KYC enabales quering the various registrar of persons in the various countries in East Africa. 
+  Visit https://developer.jengaapi.io/reference#identity-verification to see more details
+  """
```
Comment (manuelgeek) [inline lib/ex_pesa/Jenga/kyc/kyc.ex:5]: add documentation here

**Review by manuelgeek — CHANGES_REQUESTED** [review]

File: test/ex_pesa/Jenga/kyc_test.exs [hunk 3]
```diff
@@ -0,0 +1,99 @@
+defmodule ExPesa.Jenga.KYCTest do
+  @moduledoc false
```
Comment (manuelgeek) [inline test/ex_pesa/Jenga/kyc_test.exs:2]: move this file to a directory `kyc` to align with structuring system

**Review by manuelgeek — CHANGES_REQUESTED** [review]
@MidigoF  see suggestions above

File: test/ex_pesa/Jenga/kyc_test.exs (hunk 3 above)
Comment (MidigoF) [reply test/ex_pesa/Jenga/kyc_test.exs:2]: It's a single file and there's no other endpoint for KYC as opposed to others like sending money. I don't see a reason to create a folder just for that file

**Review by manuelgeek — APPROVED** [review]

**Comment by manuelgeek** [comment]
@MidigoF  rebase kindly

**Comment by MidigoF** [comment]
> @MidigoF rebase kindly

I've cherry-picked the changes instead and merged them into one commit.

**Review by manuelgeek — APPROVED** [review]
looks good @MidigoF , i think the APi has issues which is beyond us

**Pull request merged**